thanks.md

```diff
@@ -2,5 +2,22 @@
 
 In this file, you can read about where and how people use Rubberduck:
 
-- Edit this file, and tell us why you like Rubberduck. - [@username](https://github.com/username)
-- Edit this file, and tell us why you like Rubberduck. - [@username](https://github.com/username)
+<hr>
+
+I've been using VBA for as long as I can remember, but never knew anything other than a Making-It-Work coding style. I didn't give any thought to structuring my code. Needless to say, I wasn't progressing <s>very fast</s> at all. That changed early last year when I decided to improve my coding knowledge. I luckily found Rubberduck, and it’s taught me more in a year than I’d have been able to discover flying solo. Through using Rubberduck and hanging out at the the [war pond](https://chat.stackexchange.com/rooms/14929/vba-rubberducking) I’ve been able to
+- Understand the differences between `ByVal` & `ByRef` through [Code Inspections](https://github.com/rubberduck-vba/Rubberduck/wiki/Code-Inspections).
+- Learn about and successfully writing [Unit Tests](https://github.com/rubberduck-vba/Rubberduck/wiki/Unit-Testing) that allow me to make changes without fear of unintentionally breaking functionality.
+- Understand the differences between Early and Late binding.
+- Learn and use [Dependency Injection]( https://en.wikipedia.org/wiki/Dependency_injection).
+- Programming to an abstraction and not a concretion.
+- How to properly Refactor code.
+- Write Self documenting code (comments should explain the why not what).
+- Achieve consistent indentation.
+- Referencing external assemblies.
+- And a gaggle of other stuff too...
```

Comment:
IMO the list should be removed, it's already simply/vaguely summarized in the preceding sentence.

I'm expecting the average entry to read something like this (that one's an *actual* SO comment, somewhere):

> Thanks a bunch! A simple, straightforward unit testing tool for VBA is a godsend!

Or whatever fits a SO comment... or two :wink: